manifests/check.pp

```diff
@@ -107,11 +107,18 @@
 #   Set this to 'absent' to remove it completely.
 #   Default: undef
 #
+# [*content*]
```

Comment:
It was my understanding that custom adds key/values at top scope. So if you want data from custom to be in the checks scope you need the hash in custom specified as such.

Would that work? Does Puppet's $hash + $another_hash work for that?